packages/components/src/MultiSelect/MultiSelect.css

```diff
@@ -0,0 +1,38 @@
+.multiSelectContainer {
+  position: relative;
+  width: 100%;
+}
+
+.multiSelect {
+  display: flex;
+  padding-top: calc(var(--space-base) - var(--space-smallest));
+  padding-bottom: calc(var(--space-base) - var(--space-smallest));
+  padding-left: var(--space-base);
+  padding-right: var(--space-base);
+  border: var(--border-base) solid var(--color-border);
+  border-radius: var(--radius-base);
+  overflow: hidden;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.multiSelect p {
```

Comment:
Since we're using Text in the component and Text already has [truncation baked in](https://job-55182-multiselect-dropdo.atlantis.pages.dev/packages-components-src-text-text#maxlines) do we need this?